packages/php/build.sh

```diff
@@ -4,16 +4,16 @@ TERMUX_PKG_LICENSE="PHP-3.01"
 TERMUX_PKG_LICENSE_FILE=LICENSE
 TERMUX_PKG_MAINTAINER="@termux"
 # Please revbump php-* extensions along with "minor" bump (e.g. 8.1.x to 8.2.0)
-TERMUX_PKG_VERSION="8.3.10"
-TERMUX_PKG_REVISION=2
+TERMUX_PKG_VERSION="8.4.1"
+TERMUX_PKG_REVISION=1
 TERMUX_PKG_SRCURL=https://github.com/php/php-src/archive/php-${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=ef428734ff3b32bade6528b29fb0cd2dc29c5c9b1e45bb382b113a9242ab0320
+TERMUX_PKG_SHA256=490df36775b2edbefb586ecb38cb4cd781fb05ac01794a345fc1269cca10792b
 TERMUX_PKG_AUTO_UPDATE=false
 # Build native php for phar to build (see pear-Makefile.frag.patch):
 TERMUX_PKG_HOSTBUILD=true
 # Build the native php without xml support as we only need phar:
 TERMUX_PKG_EXTRA_HOSTBUILD_CONFIGURE_ARGS="--disable-libxml --disable-dom --disable-simplexml --disable-xml --disable-xmlreader --disable-xmlwriter --without-pear --disable-sqlite3 --without-libxml --without-sqlite3 --without-pdo-sqlite"
-TERMUX_PKG_DEPENDS="libandroid-glob, libandroid-support, libbz2, libc++, libcurl, libffi, libgmp, libiconv, libicu, libresolv-wrapper, libsqlite, libxml2, libxslt, libzip, oniguruma, openssl, pcre2, readline, tidy, zlib"
+TERMUX_PKG_DEPENDS="libandroid-glob, libandroid-support, libbz2, libc++, libcurl, libffi, libgmp, libiconv, libicu, libresolv-wrapper, libsqlite, libxml2, libxslt, libzip, oniguruma, openssl, pcre2, readline, tidy, zlib, postgresql"
```

Comment:
Why is `postgresql` now a dependency?
That doesn't seem to line up with any other package of PHP 8.4.x
https://gitlab.alpinelinux.org/alpine/aports/-/blob/3.21-stable/community/php84/APKBUILD
https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=php84

Postgresql should only be required for the [`php-pgsql`](https://github.com/termux/termux-packages/blob/02840d875d4d2966f222b3cca564cf519bacfc61/packages/php/php-pgsql.subpackage.sh) or [`php-apache-pgsql`](https://github.com/termux/termux-packages/blob/02840d875d4d2966f222b3cca564cf519bacfc61/packages/php/php-apache-pgsql.subpackage.sh) subpackages,
and is already defined as a subpackage dependency in both.
https://github.com/termux/termux-packages/blob/02840d875d4d2966f222b3cca564cf519bacfc61/packages/php/php-apache-pgsql.subpackage.sh#L1-L3
https://github.com/termux/termux-packages/blob/02840d875d4d2966f222b3cca564cf519bacfc61/packages/php/php-pgsql.subpackage.sh#L1-L3

Reply:
So is that only a compile time error or...

Reply:
[configure: error: Cannot find libpq-fe.h or pq library (libpq). Please specify the correct
PostgreSQL installation path with environment variables PGSQL_CFLAGS and
PGSQL_LIBS or provide the PostgreSQL installation directory.
](https://github.com/psiphon-pro19/termux-packages/actions/runs/12243611515/job/34153553965#step:6:3440)


here states [ext/pgsql/config.m4](https://github.com/php/php-src/blob/PHP-8.4.1/ext/pgsql/config.m4#L3)

Reply:
> So is that only a compile time error or...

yes, it happens compile time

Reply:
Right, then it's a build time dependency and should be specified in `TERMUX_PKG_BUILD_DEPENDS`.

Reply:
> [configure: error: Cannot find libpq-fe.h or pq library (libpq). Please specify the correct PostgreSQL installation path with environment variables PGSQL_CFLAGS and PGSQL_LIBS or provide the PostgreSQL installation directory. ](https://github.com/psiphon-pro19/termux-packages/actions/runs/12243611515/job/34153553965#step:6:3440)
> 
> here states [ext/pgsql/config.m4](https://github.com/php/php-src/blob/PHP-8.4.1/ext/pgsql/config.m4#L3)

TERMUX_PKG_DEPENDS without postgresql, will error because libpq-fe.h was not found

Reply:
I've made a testing branch and applied your changes as a patch.

It builds fine locally with `postgresql` moved to [`TERMUX_PKG_BUILD_DEPENDS`](https://github.com/termux/termux-packages/wiki/Creating-new-package#:~:text=TERMUX_PKG_BUILD_DEPENDS)
```suggestion
TERMUX_PKG_DEPENDS="libandroid-glob, libandroid-support, libbz2, libc++, libcurl, libffi, libgmp, libiconv, libicu, libresolv-wrapper, libsqlite, libxml2, libxslt, libzip, oniguruma, openssl, pcre2, readline, tidy, zlib"
TERMUX_PKG_BUILD_DEPENDS="postgresql"
```